dom-renderers/DOMRenderer.js

```diff
@@ -117,6 +117,29 @@ DOMRenderer.prototype.subscribe = function subscribe(type, preventDefault) {
 };
 
 /**
+ * Removes an EventListener of given type from the element on which it was
+ * registered.
+ *
+ * @method
+ *
+ * @param {String} type DOM event type (e.g. click, mouseover).
+ *
+ * @return {undefined} undefined
+ */
+DOMRenderer.prototype.unsubscribe = function unsubscribe(type) {
+    this._assertTargetLoaded();
+    this._target.subscribe[type] = false;
+
+    if (
+        this._target.listeners[type] || this._root.listeners[type]
+    ) {
+        var target = eventMap[type][1] ? this._root : this._target;
+        target.listeners[type] = null;
+        target.element.removeEventListener(type, this._boundTriggerEvent);
```

Comment:
@michaelobriena This might be an issue, but it is not directly related.

The problem with the proposed solution is that it will break every click (or in general every delegated event) on **every** node as soon as it is being removed on **one** node.